pkg/commands/inspect.go

```diff
@@ -113,32 +112,30 @@ func (p *printer) zipDetails() error {
 	if err := p.title("Zip"); err != nil {
 		return err
 	}
-	p.line("ZIP_NAME=%s", p.build.Config().Parameters.ZipName)
-	return p.line("ZIP_PATH=%s", p.build.Config().Paths.ZipPath)
+	if err := p.line("ZIP_NAME=%s", p.build.Config().Parameters.ZipName); err != nil {
+		return err
+	}
+	if err := p.line("ZIP_PATH=%s", p.build.Config().Paths.ZipPath); err != nil {
+		return err
+	}
+	return nil
```

Comment:
Would this be a good candidate for `firstErr()`?
```suggestion
	return firstErr(
		func() error {
			return p.line("ZIP_NAME=%s", p.build.Config().Parameters.ZipName)
		},
		func() error {
			return p.line("ZIP_PATH=%s", p.build.Config().Paths.ZipPath)
		},
	);
```

Reply:
Yes, thanks. I won't commit this as-is though with the trailing semicolon, I'll push a new commit to update it, thanks!

Reply:
😆 That's what I get for writing rust on the weekends!